Packs/CommonTypes/ReleaseNotes/3_5_12.md

```diff
@@ -0,0 +1,17 @@
+
+#### Layouts
+
+##### File Indicator
+- Updated the layout to support enrichment excluded indicators.
```

Comment:
I think we should change the note for something more meaningful, for example - Added a new Boolean field to set enrichment excluded indicators.